README.md

```diff
@@ -28,12 +28,10 @@ All other recipes should be treated as "private" and are not meant to be used in
 
 Attributes
 ----------
-| Attribute     | Default              | Description                           |
-|---------------|----------------------|---------------------------------------|
-| `build_user`  | `omnibus`            | The user to perform the Omnibus build |
-| `install_dir` | `/opt/omnibus`       | The directory to install Omnibus      |
-| `cache_dir`   | `/var/cache/omnibus` | The cache directory for Omnibus       |
-
+| Attribute     | Default                                               | Description                                              |
+|---------------|-------------------------------------------------------|----------------------------------------------------------|
+| `build_user`  | `omnibus`                                             | The user to execute Omnibus builds as                    |
+| `base_dir`    | Windows: `C:/omnibus-ruby` *nix: `/var/cache/omnibus` | The "base" directory where Omnibus will store it's data. |
 
```

Comment:
@yzl This is totally not a troll or dis: I feel lucky to have you proofreading my PRs. :heart: :ice_cream: